cmd/cluster/core/create.go

```diff
@@ -241,6 +243,14 @@ func createCommonFixture(ctx context.Context, opts *CreateOptions) (*apifixtures
 		}
 	}
 
+	if opts.DefaultDual {
+		if len(opts.AgentPlatform.APIServerAddress) <= 0 || len(opts.AgentPlatform.AgentNamespace) <= 0 {
```

Comment:
Wouldn't it be better to check the platform here since this check is to see if its any other platform other than Agent?

Reply:
As far as I saw, nop, I've been looking through the cmd and products CLIs and we bootstrap all the providers but define the options in only the specified one. I'm open for suggestions.

Reply:
Yeah I see what you're talking about. Maybe this should only look at `AgentNamespace`. The `APIServerAddress` is optional.